web-components/src/components/button/scss/settings.scss

```diff
@@ -2,7 +2,7 @@ $button__class: "#{$prefix}-button";
 
 // Default font size
 $button__font-size: rem-calc(16);
-$button__line-height: rem-calc(24);
+$button__line-height: normal;
 
```

Comment:
fixed line height was causing text to get cut off when it grew

changing to inline-flex with centre alignment will make it so it does not need to be set